SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x42664b6

how can i help troubleshoot this issue?

segfault message:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x8 pc=0x42664b6]

goroutine 1 [running]:
main.main.func1()
	/home/runner/work/aws-nuke/aws-nuke/main.go:26 +0x54
panic({0x51f1c60?, 0xa667b80?})
	/opt/hostedtoolcache/go/1.21.10/x64/src/runtime/panic.go:914 +0x21f
github.com/ekristen/aws-nuke/resources.(*CloudFormationStack).Filter(0xc0003f9b20?)
	/home/runner/work/aws-nuke/aws-nuke/resources/cloudformation-stack.go:79 +0x16
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Filter(0xc000434be0, 0xc002c09020)
	/home/runner/go/pkg/mod/github.com/ekristen/libnuke@v0.12.0/pkg/nuke/nuke.go:439 +0x398
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).runScanner(0xc000434be0, {0x6bb7bb8?, 0xc0000c7040?}, 0xc000808540, 0xc000122318)
	/home/runner/go/pkg/mod/github.com/ekristen/libnuke@v0.12.0/pkg/nuke/nuke.go:387 +0x1ee
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Scan(0xc000434be0, {0x6bb7bb8, 0xc0000c7040})
	/home/runner/go/pkg/mod/github.com/ekristen/libnuke@v0.12.0/pkg/nuke/nuke.go:412 +0x1b6
github.com/ekristen/libnuke/pkg/nuke.(*Nuke).Run(0xc000434be0, {0x6bb7bb8, 0xc0000c7040})
	/home/runner/go/pkg/mod/github.com/ekristen/libnuke@v0.12.0/pkg/nuke/nuke.go:198 +0xda
github.com/ekristen/aws-nuke/pkg/commands/nuke.execute(0xc000070280)
	/home/runner/work/aws-nuke/aws-nuke/pkg/commands/nuke/nuke.go:203 +0x1785
github.com/urfave/cli/v2.(*Command).Run(0xc000474000, 0xc000070280, {0xc00003a120, 0x9, 0x9})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/command.go:276 +0x9dd
github.com/urfave/cli/v2.(*Command).Run(0xc0004746e0, 0xc000070140, {0xc00011e0a0, 0xa, 0xa})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/command.go:269 +0xc2e
github.com/urfave/cli/v2.(*App).RunContext(0xc00003c000, {0x6bb7950?, 0xa6a9ee0}, {0xc00011e0a0, 0xa, 0xa})
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/app.go:333 +0x5db
github.com/urfave/cli/v2.(*App).Run(...)
	/home/runner/go/pkg/mod/github.com/urfave/cli/v2@v2.27.2/app.go:307
main.main()
	/home/runner/work/aws-nuke/aws-nuke/main.go:46 +0x1eb
```

cmdline:
```
aws-nuke nuke -c ~/.config/aws-nuke.yml --no-dry-run --force --force-sleep 3 --log-level trace`
```
aws-nuke version:
```
aws-nuke version v3.0.0-beta.55
```
platform:
```
$ uname -a
Linux REDACTED 6.9.2-arch1-1 #1 SMP PREEMPT_DYNAMIC Sun, 26 May 2024 01:30:29 +0000 x86_64 GNU/Linux
```

config:

```
regions:
  - "all"
  # - "global"
  #
  # - "ap-northeast-1"
  # - "ap-northeast-2"
  # - "ap-northeast-3"
  # - "ap-south-1"
  # - "ap-southeast-1"
  # - "ap-southeast-2"
  # - "ca-central-1"
  # - "eu-central-1"
  # - "eu-north-1"
  # - "eu-west-1"
  # - "eu-west-2"
  # - "eu-west-3"
  # - "sa-east-1"
  # - "us-east-1"
  # - "us-east-2"
  # - "us-west-1"
  # - "us-west-2"
  #
  # - "af-south-1"
  # - "ap-east-1"
  # - "ap-south-2"
  # - "ap-southeast-3"
  # - "ap-southeast-4"
  # - "ca-west-1"
  # - "eu-south-1"
  # - "eu-south-2"
  # - "eu-central-2"
  # - "me-south-1"
  # - "me-central-1"
  # - "il-central-1"

account-blocklist:
  # wont run against these accounts
  - "000000000000"
  - "000000000000"
  - "000000000000"
  - "000000000000"
  - "000000000000"
  - "000000000000"
  - "000000000000"

accounts:
  "000000000000":
    presets:
      - default
  "000000000000":
    presets:
      - default
  "000000000000":
    presets:
      - default
      - aws-nuke-ignore
  "000000000000":
    presets:
      - default
      - aws-nuke-ignore
    filters:
      Budget:
        - property: Name
          value: minimal budget

presets:
  aws-nuke-ignore:
    filters:
      __global__:
        - property: tag:aws-nuke
          value: ignore
  default:
    filters:
      IAMUser:
        - admin
      IAMUserPolicyAttachment:
        - admin -> AdministratorAccess
        - admin -> IAMChangeUserPassword
      IAMUserAccessKey:
        - property: UserName
          value: admin
```

--- a/resources/cloudformation-stack.go
+++ b/resources/cloudformation-stack.go
@@ -7,6 +7,7 @@ import (
 	"strings"
 	"time"
 
+	"github.com/gotidy/ptr"
 	"github.com/sirupsen/logrus"
 
 	"github.com/aws/aws-sdk-go/aws"
@@ -76,7 +77,7 @@ type CloudFormationStack struct {
 }
 
 func (cfs *CloudFormationStack) Filter() error {
-	if *cfs.stack.Description == "DO NOT MODIFY THIS STACK! This stack is managed by Config Conformance Packs." {
+	if ptr.ToString(cfs.stack.Description) == "DO NOT MODIFY THIS STACK! This stack is managed by Config Conformance Packs." {
 		return fmt.Errorf("stack is managed by Config Conformance Packs")
 	}
 	return nil

